R/app_server.R

```diff
@@ -127,13 +133,13 @@ server = function(input, output, session) {
   # get plot file
   plot_file = shiny::reactive({
     shiny::req(input$cluster_id)
-    plot_file = glue::glue("www/data/wcdemo/scanner_output/{input$cluster_id}/{input$plot_type}") # nolint
+    plot_file = glue::glue("inst/app/www/data/scanner_output/{input$cluster_id}/{input$plot_type}") # nolint
     return(plot_file)
   })
 
   # check if plots available
   plot_avail = shiny::reactive({
-    src = substring(plot_file(), 5)
+    src = substring(plot_file(), 10)
```

Comment:
If you change to using system.file above, this line will no longer work. You might be able to use
```
src = fs::path_rel(plot_file(), system.file("app", package = "tfpbrowser"))
```